server/src/guards/jwt.guard.ts

```diff
@@ -0,0 +1,27 @@
+import { Injectable, ExecutionContext } from '@nestjs/common';
+import { AuthGuard } from '@nestjs/passport';
+import { Reflector } from '@nestjs/core';
+
+@Injectable()
+export class JwtAuthGuard extends AuthGuard('jwt') {
+  constructor(private reflector: Reflector) {
+    super();
+  }
+
+  canActivate(context: ExecutionContext) {
+    // If no roles are set on the endpoint, the endpoint is public
+    const requiredRoles = this.reflector.getAllAndOverride('roles', [
```

Comment:
Not sure if I understand, but why do we need todo a role check when just using an simple auth guard